sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -129,9 +134,9 @@ private AnalyzeSentimentResult convertToTextSentimentResult(final DocumentSentim
         final SentimentConfidenceScorePerLabel confidenceScorePerLabel = documentSentiment.getDocumentScores();
 
         // Sentence text sentiment
-        final List<TextSentiment> sentenceSentimentTexts = documentSentiment.getSentences().stream()
+        final List<SentenceSentiment> sentenceSentimentTexts = documentSentiment.getSentences().stream()
```

Comment:
NIT: the `Texts` part might be misleading as it actually doesn't contain the text per se, just sentiment and scores